src/main/java/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package service.impl;
+
+import dao.UserDao;
+import java.util.List;
+import model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import service.UserService;
+
+@Service
+public class UserServiceImpl implements UserService {
+    private UserDao userDao;
+
+    @Autowired
```

Comment:
@autowired annotation is not necessary above constructor. Let's either remove it from each constructor(with injection) or use it everywhere